app/scripts/fetch_xulrunner

```diff
@@ -326,6 +326,9 @@ function modify_omni {
 	echo '.main-heading { margin-top: 2em; }' >> $file
 	# Hide Details/Permissions tabs in addon details so we only show details
 	echo 'addon-details > button-group { display: none !important; }' >> $file
+	# Hide 'Run on sites with restrictions' row and helper message
+	echo '.addon-detail-row-quarantined-domains { display: none !important; }' >> $file
+	echo '.addon-detail-help-row:has([data-l10n-id="addon-detail-quarantined-domains-help"]) { display: none !important; }' >> $file
```

Comment:
No reason for `!important` here, right?

Reply:
Right, it works just fine without it. I initially thought to add it just for good measure, in case something ever changes in the stylesheet in the future. But yeah, it's not required so I removed it.